.github/workflows/publish.yml

```diff
@@ -0,0 +1,75 @@
+name: Publish packages (PyPI, npm)
+
+on:
+  release:
+    types: [published]
+
+jobs:
```

Comment:
Let me know if you think it's a good idea to turn the `docs-update.yml` into a job of this action. This way, PyPI, npm, and docs would be published via the same action.

Reply:
OK, please do that.

Reply:
[commit 8810f4e] I didn't remove the `docs-update.yml` action because I noticed it runs on every push to `main` to update the `dev` docs. Instead, I adjusted the steps to include publishing the documentation when a release is released. Let me know if that works too.

Reply:
Actually, I can continue to use the tag creation condition instead of the release publication. Let me know if you have a preference, I am good with either.